### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by SimNee**
Following is an updated UI and formatting of the README. Hyperlinks are fixed except the link to get the jar file of our product which is not available yet.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -14,22 +40,26 @@ ifndef::env-github[]
 image::images/Ui.png[width="600"]
 endif::[]
 
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
+== Build Status
+https://travis-ci.org/se-edu/addressbook-level3[image:https://travis-ci.org/se-edu/addressbook-level3.svg?branch=master[Build Status]]
+https://ci.appveyor.com/project/damithc/addressbook-level3[image:https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true[Build status]]
```
Comment (WilliamRyank) [inline README.adoc]: delete the 2 links and change it to `https://travis-ci.org/AY1920S1-CS2103T-T13-1/main[image:https://travis-ci.org/AY1920S1-CS2103T-T13-1/main.svg?branch=master[Build Status]]`

File: README.adoc [hunk 2]
```diff
@@ -14,22 +40,26 @@ ifndef::env-github[]
...
+
+== Coverage
+https://coveralls.io/github/se-edu/addressbook-level3?branch=master[image:https://coveralls.io/repos/github/se-edu/addressbook-level3/badge.svg?branch=master[Coverage Status]]
+https://gitter.im/se-edu/Lobby[image:https://badges.gitter.im/se-edu/Lobby.svg[Gitter chat]]
```
Comment (WilliamRyank) [inline README.adoc]: delete those 2 links and add this `https://coveralls.io/github/AY1920S1-CS2103T-T13-1/main?branch=master[image:https://coveralls.io/repos/github/AY1920S1-CS2103T-T13-1/main/badge.svg?branch=master[Coverage Status]]`

File: README.adoc (hunk 1 above)
Comment (SimNee) [reply README.adoc]: Noted. Refer to next commit

File: README.adoc (hunk 2 above)
Comment (SimNee) [reply README.adoc]: Noted. Refer to next commit

**Pull request closed without merging**